public/index.html

```diff
@@ -0,0 +1,103 @@
+							<div class="mc-field-group input-group">
+								<ul>
+									<li><input type="radio" value="1" name="group[3569]" id="mce-group[3569]-3569-0"><label for="mce-group[3569]-3569-0">API beta</label></li>
+									<li><input type="radio" value="2" name="group[3569]" id="mce-group[3569]-3569-1"><label for="mce-group[3569]-3569-1">Web App beta</label></li>
+									<li><input type="radio" value="4" name="group[3569]" id="mce-group[3569]-3569-2"><label for="mce-group[3569]-3569-2">All new updates</label></li>
+								</ul>
+							</div>
+							<div class="mc-field-group">
+								<label for="mce-EMAIL">Email Address  <span class="asterisk">*</span>
+								</label>
+								<input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL">
+							</div>
+							<div class="mc-field-group">
+								<label for="mce-FNAME">First Name </label>
+								<input type="text" value="" name="FNAME" class="" id="mce-FNAME">
+							</div>
+							<div class="mc-field-group">
+								<label for="mce-LNAME">Last Name </label>
+								<input type="text" value="" name="LNAME" class="" id="mce-LNAME">
+							</div>
+
+							<div class="indicates-required"><span class="asterisk">*</span> indicates required</div>
+						</div>
+						<div id="mce-responses">
+							<div class="response" id="mce-error-response" style="display:none"></div>
+							<div class="response" id="mce-success-response" style="display:none"></div>
+						</div>    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+						<div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_1ed0ac5b8f11380ee862cf278_074bca87ce" tabindex="-1" value=""></div>
+						<input type="submit" value="Get Notified" name="subscribe" class="button">
+					</div>
+				</form>
+				<script type='text/javascript' src='//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js'></script><script type='text/javascript'>(function($) {window.fnames = new Array(); window.ftypes = new Array();fnames[0]='EMAIL';ftypes[0]='email';fnames[1]='FNAME';ftypes[1]='text';fnames[2]='LNAME';ftypes[2]='text';fnames[3]='ADDRESS';ftypes[3]='address';fnames[4]='PHONE';ftypes[4]='phone';}(jQuery));var $mcj = jQuery.noConflict(true);</script>
+				<!-- End Mailchimp Form Embed -->
+			</div>
+			<div class="social-links">
+				<a href="https://twitter.com/CancerDataLab" target="_blank">
+					<img class="social-links__icon" src="images/twitter.svg" alt="Twitter" />
+				</a>
+				<a href="https://github.com/AlexsLemonade/refinebio" target="_blank">
+					<img class="social-links__icon" src="images/github.svg" alt="Github" />
+				</a>
+				<a href="https://gitter.im/refinebio/" target="_blank">
+					<img class="social-links__icon" src="images/gitter.svg" alt="Gitter" />
+				</a>
+				<p class="social-links__credit">A project of the Childhood Cancer Data Lab, an Alex's Lemonade Stand Foundation initiative.</p>
```

Comment:
Is it better to change the line into:
```... A project of the Childhood Cancer Data Lab (CCDL), an Alex ...```
I am afraid that the term `CCDL` may be used everywhere later.